package.json

```
@@ -6,21 +6,21 @@
   "license": "UNLICENSED",
   "private": true,
   "dependencies": {
+    "@cap-js/hana": "^1.3.0",
     "@sap-cloud-sdk/http-client": "^3.2.0",
     "@sap-cloud-sdk/util": "^3.2.0",
     "@sap/audit-logging": "^5.0.0",
-    "@sap/cds": "^7.0.3",
+    "@sap/cds": "^8",
     "@sap/xb-msg-amqp-v100": "^0.9.53",
-    "@sap/xssec": "^3.6.0",
+    "@sap/xssec": "^4",
     "express": "^4",
-    "@sap/cds-hana": "^2",
     "passport": "^0.4.1"
   },
   "devDependencies": {
+    "sqlite3": "^5.1.7",
```

```suggestion
    "@cap-js/sqlite":"^1.0.1",
```
`cap-js/sqlite` to be included instead of `sqlite3`